instrumentation/httpurlconnection/testing/src/androidTest/kotlin/io/opentelemetry/instrumentation/library/httpurlconnection/InstrumentationTest.kt

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.library.httpurlconnection
+
+import io.opentelemetry.android.test.common.OpenTelemetryTestUtils
+import io.opentelemetry.instrumentation.library.httpurlconnection.HttpUrlConnectionTestUtil.executeGet
+import io.opentelemetry.instrumentation.library.httpurlconnection.HttpUrlConnectionTestUtil.post
+import io.opentelemetry.instrumentation.library.httpurlconnection.internal.HttpUrlConnectionSingletons
+import io.opentelemetry.sdk.testing.exporter.InMemorySpanExporter
+import org.junit.Assert
+import org.junit.Test
+import java.util.concurrent.Executors
+import java.util.concurrent.TimeUnit
+
+class InstrumentationTest {
+    @Test
+    fun testHttpUrlConnectionGetRequest_ShouldBeTraced() {
+        val inMemorySpanExporter = InMemorySpanExporter.create()
+        HttpUrlConnectionSingletons.setInstrumenterForTesting(OpenTelemetryTestUtils.setUpSpanExporter(inMemorySpanExporter))
+        executeGet("http://httpbin.org/get")
+        Assert.assertEquals(1, inMemorySpanExporter.finishedSpanItems.size)
```

Comment:
nit: we should use static imports for these, or use the assertj fluent style of `assertThat(...).isEqualTo(1)`. This applies to all these asserts in this PR.